src/LiveComponent/templates/Collector/live_component.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends '@TwigComponent/Collector/twig_component.html.twig' %}
+
+{% block menu %}
+    <span class="label{{ collector.components is empty ? ' disabled' }}">
+        <span class="icon">{{ source('@TwigComponent/Collector/icon.svg') }}</span>
+        <strong>Twigsss Components</strong>
```

Comment:
typo here I think

Reply:
Haha yes, I was ensuring the proper template was being rendered.

Reply:
So the file just need the {% extends %} tag right ? 

Or do we want two distinct menu entries ? (in which case i'll do another icon if you want .. loved working on the twig one ^^)

Reply:
> So the file just need the {% extends %} tag right ?

For now, yes - but I'm sort of rethinking this PR (more info below).